flang/runtime/numeric.cpp

```diff
@@ -109,7 +109,7 @@ template <typename T> inline T SetExponent(T x, std::int64_t p) {
   } else if (x == 0) {
     return 0; // 0 -> 0
   } else {
-    int expo{std::ilogb(x)};
+    int expo{std::ilogb(x) + 1};
```

Comment:
Can you also push this fix to `flang/runtime/numeric.cpp` directly into LLVM via phabricator ? We try as much as possible to keep LLVM the single point of truth for `flang/runtime`.

Reply:
Yes I will do that. Thank you again!